task-management/src/main/webapp/vue-app/taskDrawer/taskDrawerApi.js

```diff
@@ -295,8 +297,21 @@ export function urlVerify(text) {
           }
           let url = matchedText;
           if (url.indexOf('www.') === 0) {
-            url = 'http://' + url;
+            url = `http://${url}`;
```

Comment:
we could have http or https links, thus use // without specifying the protocol
```suggestion
            url = `//${url}`;
```

Reply:
Okay will update it.